Client/Application/BackgroundSyncUtil.swift

```diff
@@ -0,0 +1,97 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/
+
+import Foundation
+import BackgroundTasks
+
+class BackgroundSyncUtil {
+
+    let profile: Profile
+    let application: UIApplication
```

Comment:
I don't know why I'm confused and I feel silly asking but...... should this be passed in as a class property? Does it matter if this class is hanging on to a ref to the application? I've never actually dealt with passing the UIApplication itself around so I just don't know.